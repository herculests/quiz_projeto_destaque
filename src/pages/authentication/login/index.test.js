import React from 'react';
import Enzyme from 'enzyme';
import Login from './index';

const getDefaultComponent = (type = 'shallow') => Enzyme[type](
    <Login />,
);

describe('LoginPage', () => {
    it('Should render without error', () => {
        const component = getDefaultComponent();
        expect(component).toMatchSnapshot();
    });
});
