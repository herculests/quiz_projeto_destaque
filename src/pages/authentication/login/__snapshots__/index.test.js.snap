// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage Should render without error 1`] = `ShallowWrapper {}`;
